### DISCUSSION TITLE
FlingBehavior

### DISCUSSION THREAD

**Discussion by EMIN3M1986** [patrykandpatrick/vico#545] [Ideas]
Hello.
Add flingBehavior to scrollable as an option 
Something like video below:

Scroll always move to the selection part 

https://github.com/patrykandpatrick/vico/assets/5969144/5a6fd78e-9b7d-44ec-b1ce-fe124066d8e4

**Comment by patrickmichalik** [comment]
Hello! Thanks for the suggestion. This discussion seems to be a duplicate of #519, so I’ll be closing it. As mentioned under #519, we’ll be looking into implementing this feature.
